<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxControls">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>370</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutConrtols"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxLevel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutLevel"/>
    </widget>
   </widget>
   <widget class="QLabel" name="bloodyFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../textures/bloody_frame.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxStatus">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>210</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(78, 80, 79);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelStrength">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarHp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <zorder>bloodyFrame</zorder>
   <zorder>groupBoxLevel</zorder>
   <zorder>groupBoxControls</zorder>
   <zorder>groupBoxStatus</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLol">
    <property name="title">
     <string>Lol</string>
    </property>
   </widget>
   <addaction name="menuLol"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
